src/Libraries/RevitNodesUI/RevitDropDown.cs

```diff
@@ -735,16 +735,22 @@ public class Views : RevitDropDownBase
         protected override SelectionState PopulateItemsCore(string currentSelection)
         {
             Items.Clear();
-            //find all views in the project
-            var fec = new FilteredElementCollector(DocumentManager.Instance.CurrentDBDocument);
-            var views = fec.OfClass(typeof(View)).ToElements();
+            // find all views in the project
+            // exclude <RevisionSchedule>
+            var views = new FilteredElementCollector(DocumentManager.Instance.CurrentDBDocument)
+                .OfClass(typeof(View))
+                .Where(x => !x.Name.Contains('<'))
+                .ToList();
             
-            //there must always be at least 1 view in a Revit document, so we can exclude the empty list check
+            // there must always be at least 1 view in a Revit document, so we can exclude the empty list check
             foreach (var v in views)
             {
                 Items.Add(new DynamoDropDownItem(v.Name, v));
             }
-            return SelectionState.Restore;
+            Items = Items.OrderBy(x => x.Name).ToObservableCollection();
+            SelectedIndex = 0;
```

Comment:
By setting the selected index here you are re-setting the selection index stored in DYN file, so previous selection will no longer be saved. Is there anything specific you wanted to achieve here, otherwise returning SelectionState.Restore was good.